libs/jsonschema/schema_order.go

```diff
@@ -0,0 +1,40 @@
+package jsonschema
+
+import (
+	"slices"
+	"strings"
+)
+
+// Property defines a single property of a struct schema.
+// This type is not used in the schema itself but rather to
+// return the pair of a property name and its schema.
+type Property struct {
+	Name   string
+	Schema *Schema
+}
+
+// OrderedProperties returns the properties of the schema ordered according
+// to the value of their `order` extension. If this extension is not set, the
+// properties are ordered alphabetically.
+func (s *Schema) OrderedProperties() []Property {
+	order := make(map[string]int)
+	out := make([]Property, 0, len(s.Properties))
+	for key, property := range s.Properties {
+		order[key] = property.Order
+		out = append(out, Property{
+			Name:   key,
+			Schema: property,
+		})
+	}
+
+	// Sort the properties by order and then by name.
+	slices.SortFunc(out, func(a, b Property) int {
```

Comment:
Lets deprioritize fields that do not have the order field set.  If an author chooses not to specify an order for a field, they should be queried last because they probably don't care about the order as much for those field if they are not explict about it.

Reply:
Done.